File: Octave/0001.m
% Define the limit
limit = 1000;

% Initialize sum
sum = 0;

% Calculate the sum of multiples of 3 or 5
for i = 1:(limit-1)
  if mod(i, 3) == 0 || mod(i, 5) == 0
    sum += i;
  end
end

% Print the result
printf("The sum of all multiples of 3 or 5 below %d is: %d\n", limit, sum);

File: Visual Basic/0001.vb
Module Module1
    Sub Main()
        ' Define the limit
        Dim limit As Integer = 1000
        
        ' Calculate the sum of multiples of 3 or 5 below the limit
        Dim result As Integer = SumOfMultiples(limit)
        
        ' Print the result
        Console.WriteLine("The sum of all multiples of 3 or 5 below " & limit & " is: " & result)
    End Sub

    Function SumOfMultiples(ByVal limit As Integer) As Integer
        Dim totalSum As Integer = 0
        For i As Integer = 1 To limit - 1
            If i Mod 3 = 0 Or i Mod 5 = 0 Then
                totalSum += i
            End If
        Next
        Return totalSum
    End Function
End Module
